api/controllers/stats.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const Trials = require('../models/trial');
+const bookshelf = require('../../config').bookshelf;
+const Promise = require('bluebird');
+
+function getTrialsCount() {
+  return Trials.count();
+}
```

Comment:
When you move these methods to the models, this `getTrialsCount` becomes useless: simply use `Trials.count()` directly.